openfisca_us/tests/policy/baseline/gov/usda/snap/integration.yaml

```diff
@@ -19,14 +19,15 @@
       household:
         state_code_str: CA
   output:
-    snap: 10_020 # Maximum due to emergency allotment.
-    snap_max_allotment: 10_020 # 12 * $835 (parameter for family of 4 in contiguous US).
-    snap_expected_contribution: 5_108.4 # 17028 * 0.3 (flat parameter).
-    snap_net_income: 17_028 # 17028 - 0.
-    snap_net_income_pre_shelter: 17_028 # 24000 - 2172 - 4800.
-    snap_excess_shelter_expense_deduction: 0 # No shelter costs reported.
-    snap_standard_deduction: 2_172 # 181 (parameter for family of 4 in contiguous US) * 12.
     snap_earned_income_deduction: 4_800 # 2000 * 0.2 (flat 20% parameter) * 12.
+    snap_standard_deduction: 2_208 # 184 (parameter for family of 4 in contiguous US) * 12.
+    snap_excess_shelter_expense_deduction: 0 # No shelter costs reported.
+    snap_net_income_pre_shelter: 16_992 # 24000 - 2208 - 4800.
```

Comment:
We could use the maths here (not needed though)

Reply:
done

Reply:
Actually I changed my mind, I think keeping the values makes it easier to follow the computation tree since the values of each part go to the others, so I find putting them in comments more interpretable. We should decide and document though.